### PR TITLE
Tl 263:Replace overthere-unix.xebialabs.com with a reproducible instance in libvirt

### PR DESCRIPTION
**Pull request by eltonkevani**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -125,7 +125,7 @@ dependencies {
   compile 'commons-codec:commons-codec:1.10'
 
   // Test dependencies
-  testCompile('com.xebialabs.cloud:overcast:1.2.1') {
+  testCompile('com.xebialabs.cloud:overcast:2.5.1') {
     exclude group: 'com.xebialabs.overthere', module: 'overthere'
```
Comment (rklaren) [inline build.gradle]: Are these excludes still needed with the new overcast?

Comment (vpartington) [inline build.gradle]: This is to prevent a circular dependency. Does the new overcast still include overthere as a compile dependency?

Comment (eltonkevani) [inline build.gradle]: This one https://github.com/xebialabs/overcast/blob/master/build.gradle#L50?

Comment (rklaren) [inline build.gradle]: Looks like it can do without the extra excludes nowadays, just looked. But not a big thing.

Comment (vpartington) [inline build.gradle]: @eltonkevani Yes, that's the one.

**Pull request merged**